proxyui/src/main/webapp/resources/css/odo.css

```diff
@@ -25,3 +25,15 @@ body {
 textarea {
     resize: vertical;
 }
+
+.ui-widget-content .selectedRow {
+    color: blue;
+}
+
+.ui-jqgrid .ui-jqgrid-bdiv {
```

Comment:
fixes scrollbars appearing in chrome on jqgrid, due to 1px discrepancy between table size and table container size